script/ddgc_get_prev_contributors.pl

```diff
@@ -0,0 +1,159 @@
+#!/usr/bin/env perl
+use strict;
+use warnings;
+use FindBin;
+use lib $FindBin::Dir . "/../lib";
+use JSON;
+use DDGC;
+use HTTP::Tiny;
+use Data::Dumper;
+use Try::Tiny;
+use Net::GitHub::V3;
+use Time::Local;
+use Term::ProgressBar;
+use Date::Parse;
+use Time::Piece;
+use Time::Seconds;
+use IO::All;
+
+my $d = DDGC->new;
+
+BEGIN {
+    $ENV{DDGC_IA_AUTOUPDATES} = 1;
+}
+
+# JSON response from GH API
+my %json;
+
+# results to go into DB
+# |IA name|Repo|Issue#|title|Body|tags|created at|
+my @results;
+
+# the repos we care about
+my @repos = (
+    'zeroclickinfo-spice',
+    'zeroclickinfo-goodies',
+    'zeroclickinfo-longtail',
+    'zeroclickinfo-fathead'
+);
+
+my $token = "Add your token here";
+my $gh = Net::GitHub->new(access_token => $token);
+
+my @team  = $gh->org->team_members("52625");
```

Comment:
@jdorweiler I get an error on this line when running the script:
"Bad credentials at script/ddgc_get_prev_contributors.pl line 43."

Am I missing anything?